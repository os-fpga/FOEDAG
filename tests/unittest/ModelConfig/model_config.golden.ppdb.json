{
  "instances": [
    {
      "config_attributes": [
        {
          "I_BUF": "WEAK_KEEPER==NONE"
        },
        {
          "I_BUF": "IOSTANDARD==default"
        }
      ],
      "connectivity": {
        "I": "clock0",
        "O": "$iopadmap$clock0"
      },
      "linked_object": "clock0",
      "location": "HP_2_CC_10_5P",
      "module": "I_BUF",
      "name": "$iopadmap$dut.clock0",
      "parameters": {
        "WEAK_KEEPER": "NONE"
      },
      "properties": {}
    },
    {
      "connectivity": {
        "I": "$iopadmap$clock0",
        "O": "$auto$clkbufmap.cc:262:execute$414"
      },
      "linked_object": "clock0",
      "location": "HP_2_CC_10_5P",
      "module": "WIRE",
      "name": "AUTO CLK_BUF clock0 #0",
      "parameters": {},
      "properties": {}
    },
    {
      "connectivity": {
        "I": "$auto$clkbufmap.cc:262:execute$414",
        "O": "$auto$clkbufmap.cc:294:execute$415"
      },
      "linked_object": "clock0",
      "location": "HP_2_CC_10_5P",
      "module": "CLK_BUF",
      "name": "$auto$clkbufmap.cc:261:execute$413",
      "parameters": {},
      "properties": {}
    },
    {
      "config_attributes": [
        {
          "I_BUF": "WEAK_KEEPER==NONE"
        },
        {
          "I_BUF": "IOSTANDARD==default"
        }
      ],
      "connectivity": {
        "I": "d",
        "O": "$iopadmap$d"
      },
      "linked_object": "d",
      "location": "HR_2_6_3P",
      "module": "I_BUF",
      "name": "$iopadmap$dut.d",
      "parameters": {
        "WEAK_KEEPER": "NONE"
      },
      "properties": {}
    },
    {
      "config_attributes": [
        {
          "O_BUF": "IOSTANDARD==default"
        }
      ],
      "connectivity": {
        "I": "$iopadmap$q",
        "O": "q"
      },
      "linked_object": "q",
      "location": "HR_5_12_6P",
      "module": "O_BUF",
      "name": "$iopadmap$dut.q",
      "parameters": {},
      "properties": {}
    }
  ]
}