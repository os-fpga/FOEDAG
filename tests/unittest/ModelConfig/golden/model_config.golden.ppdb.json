{
  "instances": [
    {
      "config_attributes": [
        {
          "I_BUF": "WEAK_KEEPER==NONE"
        },
        {
          "__mapped_name__": "__parent__.u_gbox_pll_refmux_1",
          "__name__": "__parent__.__name__",
          "__optional__": "1",
          "cfg_pllref_hv_rx_io_sel": "1"
        },
        {
          "__mapped_name__": "__parent__.u_gbox_pll_refmux_1",
          "__name__": "__parent__.__name__",
          "__optional__": "1",
          "cfg_pllref_use_div": "1"
        },
        {
          "I_BUF": "IOSTANDARD==default"
        }
      ],
      "connectivity": {
        "I": "clock0",
        "O": "$iopadmap$clock0"
      },
      "linked_object": "clock0",
      "location": "HP_2_CC_10_5P",
      "module": "I_BUF",
      "name": "$iopadmap$dut.clock0",
      "parameters": {
        "WEAK_KEEPER": "NONE"
      },
      "properties": {}
    },
    {
      "connectivity": {
        "I": "$iopadmap$clock0",
        "O": "$auto$clkbufmap.cc:262:execute$414"
      },
      "linked_object": "clock0",
      "location": "HP_2_CC_10_5P",
      "module": "WIRE",
      "name": "AUTO CLK_BUF clock0 #0",
      "parameters": {},
      "properties": {}
    },
    {
      "connectivity": {
        "I": "$auto$clkbufmap.cc:262:execute$414",
        "O": "$auto$clkbufmap.cc:294:execute$415"
      },
      "linked_object": "clock0",
      "location": "HP_2_CC_10_5P",
      "module": "CLK_BUF",
      "name": "$auto$clkbufmap.cc:261:execute$413",
      "parameters": {},
      "properties": {}
    },
    {
      "config_attributes": [
        {
          "I_BUF": "WEAK_KEEPER==NONE"
        },
        {
          "__mapped_name__": "__parent__.u_gbox_pll_refmux_1",
          "__name__": "__parent__.__name__",
          "__optional__": "1",
          "cfg_pllref_hv_rx_io_sel": "1"
        },
        {
          "__mapped_name__": "__parent__.u_gbox_pll_refmux_1",
          "__name__": "__parent__.__name__",
          "__optional__": "1",
          "cfg_pllref_use_div": "1"
        },
        {
          "I_BUF": "IOSTANDARD==default"
        }
      ],
      "connectivity": {
        "I": "d",
        "O": "$iopadmap$d"
      },
      "linked_object": "d",
      "location": "HR_2_6_3P",
      "module": "I_BUF",
      "name": "$iopadmap$dut.d",
      "parameters": {
        "WEAK_KEEPER": "NONE"
      },
      "properties": {}
    },
    {
      "config_attributes": [
        {
          "O_BUF": "IOSTANDARD==default"
        },
        {
          "O_BUF": "IOSTANDARD==default",
          "__mapped_name__": "__parent__.u_HV_GBOX_BK1_A_0",
          "__name__": "__parent__.__name__"
        }
      ],
      "connectivity": {
        "I": "$iopadmap$q",
        "O": "q"
      },
      "linked_object": "q",
      "location": "HR_5_12_6P",
      "module": "O_BUF",
      "name": "$iopadmap$dut.q",
      "parameters": {},
      "properties": {}
    }
  ]
}